vispy/visuals/filters/mesh.py

```diff
@@ -572,6 +576,28 @@ def _detach(self, visual):
         super()._detach(visual)
 
 
+instanced_shading_vertex_template = shading_vertex_template.replace(
+    "$normal",
+    "mat3($instance_transform_x, $instance_transform_y, $instance_transform_z) * $normal"
+)
+
+
+class InstancedShadingFilter(ShadingFilter):
```

Comment:
Could you add a minimal docstring to this. At least a "subject line" mentioning what it does and then a small paragraph describing when it should be used over ShadingFilter and a link pointing to the shading filter for more details.

```
:class:`ShadingFilter`
```

^ That should work I think but we'll have to check the rendered HTML.

Reply:
Done!